### DISCUSSION TITLE
Publish all Morphir packages under a morphir org in npm.

### DISCUSSION THREAD

**Discussion by nwokafor-choongsaeng** [finos/morphir#139] [Proposal]
This is a proposal to publish all morphir packages under a **@morphir** npm [organization](https://docs.npmjs.com/organizations).
This pattern is already adopted by many other packages that publish multiple tools on npm like [Angular](https://www.npmjs.com/package/@angular/core).

## Benefits

- Makes finding official Morphir npm packages more predictable.
- Avoid potential name conflicts when developers contribute a morphir tooling to npm.
- Makes it easier to identify which npm packages are Morphir's official packages.

## What this means.

The *morphir-elm* package that is currently published on npm would move to *@morphir/elm* and other packages would move under *@morphir/<tool-name>*. The *morphir-elm* package would be deprecated.

**Comment by AttilaMihaly** [comment]
I love the idea!
